services/storage/requirements/_base.in

```diff
@@ -3,7 +3,6 @@
 #
 
 --constraint ../../../requirements/constraints.txt
---constraint ./constraints.txt
```

Comment:
I think this is wrong, should be kept here and the file should be left empty.

Reply:
well one can also add it if needed.